iroh/src/endpoint/quic.rs

```diff
@@ -484,3 +543,177 @@ impl QuicTransportConfig {
         self
     }
 }
+
+/// Parameters governing incoming connections
+///
+/// Default values should be suitable for most internet applications.
+// Note: used in `iroh::endpoint::connection::Incoming::accept_with`
+// This is new-typed since `quinn::ServerConfig` takes a `TransportConfig`, which we new-type as a `QuicTransportConfig`
+#[derive(Debug, Clone)]
+pub struct ServerConfig {
+    inner: quinn::ServerConfig,
+    transport: QuicTransportConfig,
+}
+
+impl ServerConfig {
+    /// Transport configuration to use for incoming connections
+    pub fn transport(&self) -> QuicTransportConfig {
+        self.transport.clone()
+    }
+
+    /// TLS configuration used for incoming connections
```

Comment:
```suggestion
    /// TLS configuration used for incoming connections.
```
😄

Reply:
I went through and added periods to all of the copy-pasted methods 😂